tests/cluster/tests/28-cluster-shards.tcl

```diff
@@ -117,7 +117,7 @@ test "Kill a node and tell the replica to immediately takeover" {
 
 # Primary 0 node should report as fail, wait until the new primary acknowledges it.
 test "Verify health as fail for killed node" {
-    wait_for_condition 50 100 {
+    wait_for_condition 1000 50 {
```

Comment:
Every 50 milliseconds, 1000 times?

Sending INFO command to valkey every 50 ms is not too often? Just wondering why you didn't keep 100 and extended the `maxtries` instead, like `wait_for_condition 500 100`.

Reply:
I always prefer shorter n ms, it maybe faster. The other reason is that we have a lot of place are doing this 50ms thing.